Source/WebKit/UIProcess/Inspector/WebPageDebuggable.h

```diff
@@ -57,10 +57,12 @@ class WebPageDebuggable final : public Inspector::RemoteInspectionTarget {
     const String& nameOverride() const final { return m_nameOverride; }
     void setNameOverride(const String&);
 
+    void detachFromPage();
+
 private:
-    Ref<WebPageProxy> protectedPage() const;
+    explicit WebPageDebuggable(WebPageProxy&);
 
-    WeakRef<WebPageProxy> m_page;
+    WeakPtr<WebPageProxy> m_page;
```

Comment:
We should replace the `#include <wtf/WeakRef.h>` with `#include <wtf/WeakPtr.h>` at the top of this file.